docs/modules/operation/pages/database-reports/database.adoc

```diff
@@ -188,30 +207,37 @@ There are a few helper methods to help create reports in {page-component-title}.
-                                                                                                                              The input parameters are prioritized. If an `snmpifdescr` is specified, it is used instead of the `snmpifname`.
-                                                                                                                              If a `snmpifdescr` is defined, it will be appended to `snmpifname`/`snmpifdescr`. +
-                                                                                                                              +
-                                                                                                                              `snmpifname`: String, the interface name of the interface; for example, `en0`. May be null. +
-                                                                                                                              `snmpifdescr`: String, the description of the interface; for example, `en0`. May be null. +
-                                                                                                                              `snmphyaddr`: String, the MAC address of the interface; for example, `005e607e9e00`. May be null. +
-                                                                                                                              As each input parameter may be null, not all of them can be null at the same time. At least one input parameter has to be defined. +
-                                                                                                                              +
-                                                                                                                              For more details, see <<ga-database-reports-interface-descriptor, Interface descriptor use>>.
+| Helper method
+| Description
+
+| getNodeOrNodeSourceDescriptor(nodeId, foreignSource, foreignId)
+| Generates a node source descriptor according to the input parameters.
+Either `node[nodeId]` or `nodeSource[foreignSource:foreignId]` is returned. +
+`nodeSource[foreignSource:foreignId]` is returned only if `foreignSource` and `foreignId` are not empty and not null.
+Otherwise, `node[nodeId]` is returned.
+
+* *nodeId*: String, the ID of the node.
+* *foreignSource*: String, the foreign source of the node. May be null.
+* *foreignId*: String, the foreign ID of the node. May be null.
+
+For more details, see <<ga-database-reports-node-source-descriptor, Node source descriptor use>>.
+
+| getInterfaceDescriptor(snmpifname, snmpifdescr, snmpphysaddr)
+| Returns the interface descriptor of a given interface; for example, `en0-005e607e9e00`.
+The input parameters are prioritized. +
+If an `snmpifdescr` is specified, it is used instead of the `snmpifname`. +
+If an `snmpifdescr` is defined, it will be appended to `snmpifname`/`snmpifdescr`.
```

Comment:
Need a check for accuracy on this.  Seems like a duplicated term.  Not sure if correct and unable to tell from the source.

Reply:
Is there some other way to verify this?

Reply:
Agree, it is appended and tracked it down to this here: 
![Screenshot 2021-08-30 at 10 54 11](https://user-images.githubusercontent.com/1095181/131313677-49a698bf-20fc-4dea-9baa-9bfe282b9476.png)

Reply:
Thanks for tracking this down!